lib/league_statistics.rb

```diff
@@ -0,0 +1,36 @@
+require 'csv'
+require_relative './stat_tracker'
+require_relative './game'
+require_relative './team'
+require_relative './game_team'
+require_relative './season_statistics'
+
```

Comment:
I wonder what the most efficient way to pass initialize method through all child classes would look like?